### PR TITLE
chore: better CTA

### PR DESCRIPTION
**Pull request by tianzhou**
Replace newsletter subscription with CTA

CleanShot 2022-08-12 at 10 21 47

CleanShot 2022-08-12 at 10 21 58

### CODE REVIEW

File: components/ActionSectionBottom.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="bg-indigo-700 rounded-lg">
+    <div class="max-w-7xl mx-auto py-12 px-4 sm:px-12">
+      <h2
+        class="space-y-1 text-3xl font-semibold tracking-tight text-white sm:text-4xl"
+      >
+        <span class="block">Review, change, version and restore.</span>
+        <span class="block">Database DevOps with ease.</span>
```
Comment (boojack) [inline components/ActionSectionBottom.vue:8]: i18n?

File: components/ActionSectionBottom.vue [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    moduleName: {
+      required: true,
+      type: String,
+    },
+  },
+  setup(props) {
+    const { app } = useContext();
+    const analytics = ref<Metric>();
+
+    const currentLocale = computed(() => app.i18n.locale);
```
Comment (boojack) [inline components/ActionSectionBottom.vue:65]: This variable is unused.

**Comment by tianzhou** [comment]
PTAL

**Review by boojack — APPROVED** [review]
LGTM

**Review by d-bytebase — APPROVED** [review]

**Pull request merged**